backend/src/services/group/group.service.ts

```diff
@@ -51,6 +52,43 @@ class Group {
 
     return newGroup;
   }
+
+  public async update(
+    id: string,
+    { name, tenantId, workersIds, permissionsIds }: EAMGroupCreateRequestDto,
+  ): Promise<EAMGroupCreateResponseDto | null> {
+    const group = await this.#groupRepository.getGroupById(id);
+
+    if (!group) {
+      return null;
+    }
+    if (name) {
+      const groupByName = await this.#groupRepository.getGroupByNameAndTenant(
+        name,
+        tenantId,
+      );
+
+      if (groupByName) {
+        throw new InvalidGroupNameError();
+      }
+
+      group.setName(name);
+    }
+
+    group.setPermissionsIds(permissionsIds);
```

Comment:
Maybe this help you.?
```suggestion
    permissionsIds && group.setPermissionsIds(permissionsIds);
```